sound/soc/codecs/hdac_hdmi.c

```diff
@@ -1973,12 +1998,24 @@ static int hdac_hdmi_get_spk_alloc(struct hdac_device *hdev, int pcm_idx)
 	return port->eld.info.spk_alloc;
 }
 
+static int icl_pin2port_map[] = {4, 6, 8, 10, 11};
```

Comment:
should this be const ?

Reply:
I wasn't able to figure out how those values were determined, can you share a private email on what parts of the hardware spec you are looking at?